templates/nginx-conf.j2

```diff
@@ -47,6 +47,10 @@ http {
     sendfile        on;
     #tcp_nopush     on;
 
+    {% for http_conf_line in nginx_proxy_conf_http -%}
```

Comment:
But since nginx doesn't mind the whitespace, I'll settle with 2x extra chars after the addition of the `-` here.